Src/CSharpier/DocPrinter.cs

```diff
@@ -71,53 +77,53 @@ private Indent MakeIndent(Indent indent, Options options)
                 }
             }
 
-            flushSpaces();
+            FlushSpaces();
 
-            void addTabs(int count)
+            void AddTabs(int count)
```

Comment:
Did you also port these local functions from prettier? It might be more C# like to make them instance methods and extract this into its own class. What do you think?

Reply:
A good chunk of this was ported right from prettier. I've been debating doing some major refactoring to it because I always get lost inside of it. But I think first I should figure out if any of the other features from prettier are needed.